### ISSUE TITLE
Sorry to bother. --Activating

### ISSUE THREAD

**Issue by cc4digital** [lightSoulDev/vs-code-comment-sections#1]
Within VS code after installation, how to you activate?  
is it alt / or just when you type /** ???
Thanks in advance
Chuck

**Comment by lightSoulDev** [comment]
Hi!, It's in a FEATURE CONTRIBUTIONS tab:
coolcommentsections.generateComment: Ctrl+Alt+/
just write text, select and press the shortcut.

**Comment by cc4digital** [comment]
Sorry about that.  Yeah, I had VS code comments set to Ctrl + Alt + /.  I then looked for your program within keyboard preferences, but did not see it under "coolcommentsections.generateComment: " 

Anyway, I have now updated coolcommentssections keyshortcut to shift+ctrl + alt +/, and all is working as expected.

Thanks for the reply.

Lastly, this is a nice program and a nice idea.  
![sample](https://github.com/lightSoulDev/vs-code-comment-sections/assets/10839846/79571abc-10f5-4ca0-8396-b1cb196affd0)

Thanks, Chuck
